src/DevBetterWeb.Web/Pages/Admin/UserReports/ReportsDatesModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace DevBetterWeb.Web.Pages.Admin.UserReports
+{
+  public class ReportsDatesModel
+  {
+    public DateTime? StartDate { get; set; } 
+    public DateTime? EndDate { get; set; } 
+
+    public ReportsDatesModel()
+    {
+      // for model binding
+    }
+
+    public ReportsDatesModel(DateTime startDate, DateTime endDate)
+    {
+      StartDate = startDate;
+      EndDate = endDate;
```

Comment:
If you want end dates to be inclusive of all times on that date, you could add logic here to make enddate be date 23:59:59 on that date instead of 00:00:00. Then you wouldn't need to change things elsewhere. To make that clear you could rename this to something like InclusiveEndDate for instance.